src/test/java/org/isf/medicalstockward/test/Tests.java

```diff
@@ -24,10 +24,12 @@
 import static org.assertj.core.api.Assertions.assertThat;
 import static org.assertj.core.api.Assertions.assertThatThrownBy;
 import static org.assertj.core.data.Offset.offset;
+import static org.junit.jupiter.api.Assertions.assertThrows;
```

Comment:
why add a new `throws` type test when `assertThatThrownBy` is already used in the file?

Reply:
yes, please check now, thanks!